### ISSUE TITLE
Response with matched country!

### ISSUE THREAD

**Issue by shahenazmonia** [FACK1/Countries-autoComplete#26]
The response should be with the matched country, not all countries! 
https://github.com/FACK1/Countries-autoComplete/blob/4dd217a462dc2d6c0b4f8f9744226206f99dfc44/src/handlers.js#L60

You should filter the json file and return the matched country

**Comment by Hadeel-Salamin** [comment]
yeah but we did not have time to fix that yesterday, will work on that today if we had time.
